website/client/components/header/index.vue

```diff
@@ -175,11 +175,17 @@ export default {
         this.$root.$emit('bv::show::modal', 'create-party-modal');
       }
     },
-    showPartyMembers () {
+    async showPartyMembers () {
+      // If party, retrieve the full party details to allow permissions checks.
+      if (this.user.party._id) {
+        await this.$store.dispatch('party:getParty', true);
+        this.party = this.$store.state.party.data;
```

Comment:
is this.party defined only here?

Reply:
you're right, @titchimoto could you add `party` to the data object?